requirements.txt

```diff
@@ -14,6 +14,8 @@ django-appconf==1.0.1
 django-bootstrap3==5.4.0
 celery==3.1.18
 redis==2.10.3
+git+https://github.com/mitodl/django-guardian@app_label_none#egg=django-guardian-dev
```

Comment:
I still would prefer the hash of HEAD of that branch so we can still update that branch as ended without breaking existing deployments, but a comment of what branch it is at, would also be helpful